docs/usage/two-phase-transfer.md

```diff
@@ -17,13 +17,22 @@ funds in stages:
 cannot be used by either the payer or payee. Only the
 [`debits_pending`](../reference/accounts.md#debits_pending) and
 [`credits_pending`](../reference/accounts.md#credits_pending) fields are
-modified on the relevant accounts at this point.
+increased on the relevant accounts at this point.
 
 2. Later, the application creates another transfer — with either of
    the following flags:
 * [`post_pending_transfer`](../reference/transfers.md#flagspost_pending_transfer): Move all (or part) of the reserved funds to the pending transfer's destination.
 * [`void_pending_transfer`](../reference/transfers.md#flagsvoid_pending_transfer): Revert all of the reserved funds to the original account.
 
+Once the pending transfer is posted or voided, the
+[`debits_pending`](../reference/accounts.md#debits_pending) and
+[`credits_pending`](../reference/accounts.md#credits_pending) fields
+on the relevant accounts are decreased by the
+[`amount`](../reference/transfer.md#amount) of the transfer. And the
+[`debits_posted`](../reference/accounts.md#debits_posted) and
+[`credits_posted`](../reference/accounts.md#credits_posted) fields on
+the relevant accounts are increased by that same amount.
```

Comment:
The pending part is right, but the posted part needs more detail.
Transfers can be partially posted, in which case they may increase but only a subset of the pending amount.
Or when voided, then the posted debits/credits don't change at all.

Reply:
I added a section on "Posting a subset of pending `amount`" which maybe addresses your first concern. But I don't understand the latter.

Reply:
How about something like this?

When the pending transfer is resolved (posted or voided),
[`debits_pending`](../reference/accounts.md#debits_pending) and
[`credits_pending`](../reference/accounts.md#credits_pending) fields
on the respective accounts are decreased by the
[`amount`](../reference/transfer.md#amount) of the _pending_ transfer.

When the pending transfer is posted,
[`debits_posted`](../reference/accounts.md#debits_posted) and
[`credits_posted`](../reference/accounts.md#credits_posted) fields on
the respective accounts are increased by the _posting_ transfer's `amount`
(which cannot exceed the pending amount).

When the pending transfer is voided,
[`debits_posted`](../reference/accounts.md#debits_posted) and
[`credits_posted`](../reference/accounts.md#credits_posted) are not modified.

Reply:
Looks good, added a short phrase to clarify posted amount need not equal pending amount either